data/pigui/modules/info-view/03-econ-trade.lua

```diff
@@ -175,9 +175,10 @@ end
 -- Gauge bar for used/free cabins
 local function gauge_cabins()
 	local player = Game.player
-	local cabins_total = player:GetEquipCountOccupied("cabin")
-	local cabins_free = player.cabin_cap or 0
-	local cabins_used = cabins_total - cabins_free
+	local cabins_free = Passengers.CountFreeCabins(player)
+	local cabins_used = Passengers.CountOccupiedCabins(player)
+	local cabins_total = cabins_used + cabins_free
```

Comment:
TODO: probably need to count berths, not cabins, here.

Reply:
Semantically, this considers a "cabin" to be an individual berth within a cabin slot equipment. Will rename as appropriate.